.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+  push:
+
+jobs:
+  e2e-kubernetes:
+    name: Kuebrnetes E2E tests
+    runs-on: ubuntu-18.04
+    env:
+      E2E_MINIKUBE_DRIVER: docker
```

Comment:
In travis, minikube driver was set to `none`. I have found that `docker` plays better with `manusa/actions-setup-minikube@v2.0.1`, and removes the need for sudo, which is always a plus.

However `none` can probably be made to work as well, if it is preferred for other reasons.

Reply:
No idea why it was set to none, we can move it to docker I believe